src/librustdoc/html/render.rs

```diff
@@ -2443,8 +2443,10 @@ fn item_enum(w: &mut fmt::Formatter, cx: &Context, it: &clean::Item,
             write!(w, "</code></span></span>")?;
             document(w, cx, variant)?;
 
-            use clean::{Variant, StructVariant};
-            if let clean::VariantItem( Variant { kind: StructVariant(ref s) } ) = variant.inner {
+            use clean::{Variant, VariantKind};
+            if let clean::VariantItem(
+                Variant { kind: VariantKind::Struct(ref s) } ) = variant.inner
+            {
```

Comment:
nit: this would be easier for me to read:
```rust
            if let clean::VariantItem(Variant { kind: VariantKind::Struct(ref s) }) =
                variant.inner
            {
```